slash/react/src/Form/Date/DateInput.tsx

```diff
@@ -15,15 +15,24 @@ const DateInput = forwardRef<HTMLInputElement, Props>(
     return (
       <Field
         {...otherProps}
-        renderInput={({ id, classModifier, ariaInvalid, errorId }) => (
-          <Date
-            id={id}
-            classModifier={classModifier}
-            ref={inputRef}
-            aria-describedby={errorId}
-            aria-invalid={ariaInvalid}
-            {...otherProps}
-          />
+        renderInput={({
+          id,
+          classModifier,
+          ariaInvalid,
+          errorId,
+          children: childrenField,
+        }) => (
+          <>
+            <Date
+              id={id}
+              classModifier={classModifier}
+              ref={inputRef}
+              aria-describedby={errorId}
+              aria-invalid={ariaInvalid}
+              {...otherProps}
+            />
+            {childrenField}
+          </>
         )}
       >
         {children}
```

Comment:
le children qu'on a ici sert à quoi du coup ? j'ai du mal à comprendre la différence entre les deux

Reply:
Le children passé à `Field` permet d'appliquer des styles au field et le `childrenField` est le children passer dans les props de `renderInput`.
In fine c'est le même mais le passé à `Field` plutôt que directement au composant du `renderInput`permet de déterminer des styles en plus.
Comme le `Field` est un refactor, peut etre que l'on peut faire mieux